src/javascripts/modules/ExpandListItem.js

```diff
@@ -0,0 +1,19 @@
+export default class ExpandListItem {
+  constructor(el) {
+    this.el = el
+    this.tag = el.parentNode
```

Comment:
When refactoring to use `aria-expanded` with `aria-controls`, you can save a reference to the label/text element in the constructor, instead of using the tag:
```
this.label = document.getElementById(this.el.getAttribute('aria-controls'))
```